concourse/pipelines/create-deployer.yml

```diff
@@ -0,0 +1,299 @@
+---
```

Comment:
should have used `git mv` instead of deleting old file and adding new one. It would be easier to see what was added/changed.